test/spec/app_spec_helpers.rb

```diff
@@ -101,8 +101,41 @@ def slack_login
     sleep 3
   end
 
+  def wait_for_selector(selector, type = :css, timeout = 30)
+    @wait = Selenium::WebDriver::Wait.new(timeout: timeout)
+    element = @wait.until { @driver.find_element(type, selector) }
+    element
+  end
+
+  def wait_for_selector_list(selector, type = :css, timeout = 30)
+    @wait = Selenium::WebDriver::Wait.new(timeout: timeout)
+    elements = @wait.until { @driver.find_elements(type, selector) }
+    elements
+  end
+
+  def wait_txt_change(txt, selector, type = :css)
+    count = 0
+    begin
+      count = count + 1
+      el = wait_for_selector(selector, type)
+      sleep 1
+    end while (el.text == txt and count < 10)
+    el.text
+  end
+
+  def wait_size_change(s, selector, type = :css)
+    count = 0
+    begin
+      count = count + 1
+      el = wait_for_selector_list(selector, type)
+      sleep 1
+    end while (s == el.size and count < 10)
```

Comment:
Rename to `wait_for_size_change` and add `count` parameter with default = 10

Reply:
Ok